pkg/piecestore/psserver/psdb/psdb.go

```diff
@@ -103,6 +103,93 @@ func (db *DB) Migration() *migrate.Migration {
 					)`,
 				},
 			},
+			{
+				// some databases may have already this done, although the version may not match
+				Description: "Optimize storagenode tables",
+				Version:     1,
+				Action: migrate.Func(func(log *zap.Logger, db migrate.DB, tx *sql.Tx) error {
+					v1sql := migrate.SQL{
+						`ALTER TABLE bandwidth_agreements RENAME TO bandwidth_agreements_old`,
```

Comment:
No need to create new table. Just add columns to existing one. Something like this:
```
ALTER TABLE bandwidth_agreements ADD uplink BLOB;
ALTER TABLE bandwidth_agreements ADD serialnum TEXT NOT NULL DEFAULT "";
ALTER TABLE bandwidth_agreements ADD total INT(10);
ALTER TABLE bandwidth_agreements ADD maxsize INT(10);
...
```
Then iterate over `bandwidth_agreements` and just update new columns.

Reply:
Done